run_test.sh

```diff
@@ -65,12 +68,35 @@ do
         --verbose)
             VERBOSE=$arg
             ;;
+        --self)
+            SELF=1
+            ;;
         *=*)
             eval $(echo $arg | cut -d= -f1)=\"$(echo $arg | cut -d= -f2-)\"
             ;;
     esac
 done
 
+if [[ -n "${SELF}" ]]; then
+    # Ensure schema-salad-tool command
+    if [[ ! -x $(command -v schema-salad-tool) ]]; then
+        if [[ ! -d ./schema_salad ]]; then
+            echo >&2 "You need: git submodule update --init"
+            exit 1
+        fi
+        # Install schema_salad to validate cwl files.
+        cd ./schema_salad && pip install . --quiet && cd ..
+    fi
+    # This is how CWL should be written.
+    DEFINITION=./schema_salad/schema_salad/tests/test_schema/CommonWorkflowLanguage.yml
```

Comment:
That file is two years out of date, why not use the one in this repository?

Reply:
Ah! I didn't see it. I will fix it to see the `./v1.0/CommonWorkflowLanguage.yml`!!